### PR TITLE
Updated CortanaSample instructions.

### PR DESCRIPTION
**Pull request by asimonov-msft**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/winobjc-samples/21)

### CODE REVIEW

**Review by andrejo-msft — APPROVED** [review]

**Review by kellymor — APPROVED** [review]

File: Scenarios/Cortana/README.md [hunk 1]
```diff
@@ -7,6 +7,17 @@ These are the basic steps to add voice command functionality and integrate Corta
 1. **Create a Voice Command Definition (VCD) file and register it when the app is launched.** The VCD file is an XML document that defines all the spoken commands that the user can say to initiate actions or invoke commands when activating your app. For more information, see [VCD elements and attributes v1.2](https://msdn.microsoft.com/library/windows/apps/dn706593).
 2. **Handle the activation-by-voice command.** You may want to handle launch differently depending on the method by which a user launched your app.
 
+## Running the Sample
+1. Open the CortanaSample-WinStore10.sln Visual Studio solution.
+2. Ensure that the project includes the CortanaSample\CortanaSampleCommands.xml VCD file. If it does not, add it by selecting **Project** > **Add Existing Item**.
+3. Build and deploy the application to your phone or PC.
+4. Start the sample app for the first time, which should automatically register it with Cortana. **Close it immediately.**
```
Comment (philnach) [inline Scenarios/Cortana/README.md]: As supposed to saying it should automatically register...  How about:

4. Start the sample app in order to register the VCD file with Cortana. **Close sample app.**  #Closed

File: Scenarios/Cortana/README.md [hunk 2]
```diff
@@ -7,6 +7,17 @@ These are the basic steps to add voice command functionality and integrate Corta
...
+5. Open **Cortana** and try making some of the pre-configured queries. These can be performed via voice or by typing them into Cortana's search bar:
```
Comment (philnach) [inline Scenarios/Cortana/README.md]: Open **Cortana** and try some of the commands and phrases defined in the VCD.  The commands can be... #Closed

File: Scenarios/Cortana/README.md [hunk 3]
```diff
@@ -7,6 +7,17 @@ These are the basic steps to add voice command functionality and integrate Corta
...
+    * "Cortana Sample, show trip to London."
+    * "Show trip to Long Island in Cortana Sample."
+    * "Show my Cortana Sample trip to Rio."
+6. The sample app should open automatically, and reflect the query made to Cortana.
```
Comment (philnach) [inline Scenarios/Cortana/README.md]: 6. The sample app should open automatically, and reflect the commands and phrases provided to Cortana.  #Closed

File: Scenarios/Cortana/README.md [hunk 4]
```diff
@@ -224,7 +216,7 @@ When Cortana is listening, here are some of the voice commands that work in the
 * "Cortana Sample, show trip to London"
 
 Infix and suffix forms are also supported:
-* "Show trip to Rioin Cortana Sample"
+* "Show trip to Rioin in Cortana Sample"
```
Comment (philnach) [inline Scenarios/Cortana/README.md]: It should be Rio and not Rioin #Closed

**Review by philnach — CHANGES_REQUESTED** [review]
Minor nits

**Review by philnach — APPROVED** [review]

**Pull request merged**